RecoTracker/LSTCore/src/alpaka/Quintuplet.h

```diff
@@ -0,0 +1,3237 @@
+                                                      float& innerInvRadiusMin,
+                                                      float& innerInvRadiusMax,
+                                                      float& bridgeInvRadiusMin,
+                                                      float& bridgeInvRadiusMax,
+                                                      float& outerInvRadiusMin,
+                                                      float& outerInvRadiusMax) {
+    float innerInvRadiusErrorBound = 0.1512f;
+    float bridgeInvRadiusErrorBound = 0.1781f;
+    float outerInvRadiusErrorBound = 0.1840f;
+
+    if (innerRadius > 2.0f / (2.f * k2Rinv1GeVf)) {
+      innerInvRadiusErrorBound = 0.4449f;
+      bridgeInvRadiusErrorBound = 0.4033f;
+      outerInvRadiusErrorBound = 0.8016f;
+    }
+
+    innerInvRadiusMax = (1.f + innerInvRadiusErrorBound) / innerRadius;
+    innerInvRadiusMin = alpaka::math::max(acc, 0.f, (1.f - innerInvRadiusErrorBound) / innerRadius);
+
+    bridgeInvRadiusMax = (1.f + bridgeInvRadiusErrorBound) / bridgeRadius;
+    bridgeInvRadiusMin = alpaka::math::max(acc, 0.f, (1.f - bridgeInvRadiusErrorBound) / bridgeRadius);
+
+    outerInvRadiusMax = (1.f + outerInvRadiusErrorBound) / outerRadius;
+    outerInvRadiusMin = alpaka::math::max(acc, 0.f, (1.f - outerInvRadiusErrorBound) / outerRadius);
+
+    return checkIntervalOverlap(innerInvRadiusMin, innerInvRadiusMax, bridgeInvRadiusMin, bridgeInvRadiusMax);
+  };
+
+  template <typename TAcc>
+  ALPAKA_FN_ACC ALPAKA_FN_INLINE bool matchRadiiBBBBE(TAcc const& acc,
+                                                      const float& innerRadius,
+                                                      const float& bridgeRadius,
+                                                      const float& outerRadius,
+                                                      const float& innerRadiusMin2S,
+                                                      const float& innerRadiusMax2S,
+                                                      const float& bridgeRadiusMin2S,
+                                                      const float& bridgeRadiusMax2S,
+                                                      const float& outerRadiusMin2S,
+                                                      const float& outerRadiusMax2S,
+                                                      float& innerInvRadiusMin,
+                                                      float& innerInvRadiusMax,
+                                                      float& bridgeInvRadiusMin,
+                                                      float& bridgeInvRadiusMax,
+                                                      float& outerInvRadiusMin,
+                                                      float& outerInvRadiusMax) {
+    float innerInvRadiusErrorBound = 0.1781f;
+    float bridgeInvRadiusErrorBound = 0.2167f;
+    float outerInvRadiusErrorBound = 1.1116f;
```

Comment:
If we have slides, I suggest to add links to corresponding documentation. If pynb is not available, pdf presentation is already better than nothing.

Reply:
Maybe this slide could be useful for T5 radius matching
http://uaf-10.t2.ucsd.edu/~bsathian/SDL_slides/20210513-t5LessonsLearned.pdf

Reply:
Thanks for the direct links.
I suggest to copy these to a CMS-accessible place (perhaps on indico). uaf links tend to die; in case these are to be added inline in the code

Reply:
I download and uploaded the slides in one of our LST meeting indico links. Maybe this page stays longer?
https://indico.cern.ch/event/1410988/contributions/5931044/attachments/2891001/5067597/20210513-t5LessonsLearned.pdf
https://indico.cern.ch/event/1410988/contributions/5931044/attachments/2891001/5067598/20210316-t5NewCuts.pdf